docs/guides/other-guides/monitor-ngrok/monitor-ngrok.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Observe your API traffic with ngrok and Datadog
+sidebar_label: Monitor with Datadog
+sidebar_position: 3
+toc_max_heading_level: 3
+---
+
+## Introduction
+
+This guide explains how to monitor your API or web app with ngrok by viewing traffic reports, using error request replays, and exporting logs and events to an external dashboard.
+
+Whether you're an existing ngrok user looking to make your API more robust or a new user wondering whether ngrok meets your needs, this tutorial will demonstrate monitoring in detail.
+
+## Prerequisites
+
+This guide assumes you already have [an ngrok account](https://dashboard.ngrok.com/signup) (a free account is sufficient) and have [Docker installed](https://docs.docker.com/get-started/get-docker) on your computer.
+
+You'll use the [ngrok sample API](https://github.com/ngrok-samples/api-demo) in this tutorial. Even if you have an existing API you want to monitor, you can test the sample before making changes to your real API. If you have no interest in running the sample app and immediately want to work on your live app, jump ahead to the [Monitor your API](#monitor-your-api) section.
+
+## Start the sample API to monitor
+
+Let's start the sample API. Open a terminal and run the commands below.
+
+```sh
+docker network create ngrokTest
+
+docker run --init --platform=linux/amd64 --rm -p 4000:80 --network=ngrokTest --name="api" -e PORT=80 joelatngrok/api-demo
+```
+
+This command runs the Docker sample API container, exposing the API locally on port 4000, names the Docker container `api`, and removes the container with `--rm` upon exiting. You can now browse to http://localhost:4000 to test the API.
+
+:::note
+The API uses a named network, `ngrokTest`, so that in the next section, you can start the ngrok Agent on the same network as the API.
+:::
+
+![Simple web API](./img/simple-web-api.png)
+
+## Publish your API with ngrok
+
+Now that the API is running locally, you can expose it on a public URL, using the [Docker image for the ngrok agent](https://hub.docker.com/r/ngrok/ngrok).
+
+- Replace the authentication token in the command below with your token from the [ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken), and run the command in a new terminal.
+
+  ```sh
+  docker run -it --rm --platform=linux/amd64 --network=ngrokTest -e NGROK_AUTHTOKEN=Y0urS3cr3tK3y ngrok/ngrok:3.22.0-alpine-amd64 http http://api
+  ```
+
+  This command starts the ngrok agent locally and connects it to the API running on the `api` container.
+
+- Browse to the URL labeled `Forwarding`, which should look like this in your terminal: `https://eb45-79-127-145-72.ngrok-free.app`.
+- You can now see your request going from the browser to the ngrok agent you're running in one terminal window and then to the API in your other terminal window.
+
+![Published API](./img/api-gent.png)
```

Comment:
What I'd prefer to do here is cut all this and instead use the change I requested above, which points folks to the previous API gateway guides, so that they already have the API demo app running and their whole API gateway configured.